projects/transformers/experiments/finetuning.py

```diff
@@ -129,6 +129,38 @@
     model_name_or_path="/mnt/efs/results/pretrained-models/transformers-local/bert_100k",  # noqa: E501
 )
 
+# the name 'simple' is in reference to the paper
```

Comment:
This may be more of a nit pick style comment, but I believe in using proper grammar and punctuation in comments, especially longer ones. I believe it makes it easier to read. This one could be formatted as 
```
# The name 'simple' is in reference to the paper "On the stability of finetuning BERT"
# where they propose a "simple but hard to beat" approach
#       https://openreview.net/pdf?id=nzpLWnVAyah
# 
# How to decided num_train_epochs: They say rte for 20 epochs is good ...
```
Notice how I try and use the full line length as well. There should be an extension in vscode to help automatically format comments for line length.